examples/ts-dev-gauge-googlecharts/src/main.ts

```diff
@@ -55,9 +57,9 @@ Spotfire.initialize(async (mod) => {
         // Add marking highlight using the marking color, if marking is enabled.
         let marking = await dataView.marking();
         let gauges = gauge.element.getElementsByTagName("td");
-        rows.forEach((row, index) => {
+        categoryRoot.leaves().forEach((leaf, index) => {
             gauges[index].style.background =
-                row.isMarked() && marking
+                leaf.rows()[0]?.isMarked() && marking
```

Comment:
This did not become more reader friendly. I would have prefered:
```suggestion
                marking && leaf.rows().filter(r => r.isMarked()).length
```